Gemfile

```diff
@@ -3,10 +3,7 @@ source 'https://rubygems.org'
 gem 'jwe', '~> 0.4.0'
 gem 'pry'
 gem 'rufus-scheduler'
-gem 'sinatra'
```

Comment:
Why do you remove Sinatra and Thin? (not sure about tzinfo)
This seems to be a Sinatra app (take a look at the file `start.sh`), without the gem and without a server (Thin) effectively this will break badly.